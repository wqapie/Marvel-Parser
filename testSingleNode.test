# Find the path for a single node with no edges.

LoadGraph oneNode oneNode.tsv
ListChildren oneNode Robin.Wang
FindPath oneNode Robin.Wang Robin.Wang