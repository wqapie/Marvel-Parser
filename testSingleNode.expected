# Find the path for a single node with no edges.

loaded graph oneNode
the children of Robin.Wang in oneNode are:
path from Robin.Wang to Robin.Wang: